certbot/storage.py

```diff
@@ -78,6 +78,9 @@ def write_renewal_config(o_filename, n_filename, target, relevant_data):
         if k not in relevant_data:
             del config["renewalparams"][k]
 
+    if "renew_before_expiry" not in config:
+        config.initial_comment = ["renew_before_expiry = 30 days"]
+
```

Comment:
That popped in my head as well, but I thought that since it's a comment, it doesn't make much of a difference.

Your comment solves the dilemma. I'll add a commit in a few.